### PR TITLE
storage: Move NFS management into new details page

### LINKED ISSUE #8561: storage: NFS, using the checkbox to edit a share is not intuitive
In the [usability testing of NFS](
https://github.com/cockpit-project/cockpit/wiki/Usability-test-NFSclient
), we identified that all four participants struggled with using the checkmark button to trigger the edit mode of the list.

### LINKED ISSUE #8565: storage: unclear difference between unmount and delete
In the [usability testing of NFS](
https://github.com/cockpit-project/cockpit/wiki/Usabilitytest-NFSclient
), we identified that all three out of four participants had issue distinguishing between unmount and delete.

### PR DESCRIPTION
**Pull request by mvollmer**
This gets rid of the confusing "arm" button.

Fixes #8561
Fixes #8565

### CODE REVIEW

**Comment by mvollmer** [comment]
Layout of usage bar plus text is wrong.

**Comment by mvollmer** [comment]
> Layout of usage bar plus text is wrong.

Fixed.

File: pkg/storaged/client.js [hunk 1]
```diff
@@ -507,7 +507,9 @@
             mount_entry: mount_entry,
             unmount_entry: unmount_entry,
             stop_and_unmount_entry: stop_and_unmount_entry,
-            stop_and_remove_entry: stop_and_remove_entry
+            stop_and_remove_entry: stop_and_remove_entry,
+
```
Comment (martinpitt) [inline pkg/storaged/client.js]: nitpick: empty line

**Comment by mvollmer** [comment]
@andreasn, could you have a look?

**Comment by mvollmer** [comment]
> The design had only path-on-server in the breadcrumb bar, and now it has path-on-server + "on" + local mount point

I'll change this.

**Comment by mvollmer** [comment]
> I'll change this.

Done.

**Comment by martinpitt** [comment]
Remaining test failures are unrelated.

**Comment by martinpitt** [comment]
Screenshots for release notes:

nfs-list
nfs-details

**Pull request merged**